### PR TITLE
Integrate Gramine Example to local ci

### PR DESCRIPTION
**Pull request by anjalirai-intel**

### CODE REVIEW

File: ci/linux-direct-centos8-docker-gcc-release.jenkinsfile [hunk 1]
```diff
@@ -44,9 +44,9 @@ node(node_label) {
                 load '.ci/lib/config-release.jenkinsfile'
 
                 load '../ci/stage-build-nosgx.jenkinsfile'
-                load '../ci/stage-test.jenkinsfile'
                 load '../ci/stage-test-direct.jenkinsfile'
                 load '../ci/stage-test-sandstone.jenkinsfile'
+                load '../ci/stage-test.jenkinsfile'
```
Comment (jinengandhi-intel) [inline ci/linux-direct-centos8-docker-gcc-release.jenkinsfile:49]: what is this change for? generally unrelated refactoring should be done in a separate PR.

File: ci/linux-direct-centos8.2-gcc-release.jenkinsfile [hunk 2]
```diff
@@ -38,11 +38,10 @@ node(node_label) {
             load '.ci/lib/config-release.jenkinsfile'
 
             load '../ci/stage-build-nosgx.jenkinsfile'
-            load '../ci/stage-test.jenkinsfile'
             load '../ci/stage-test-direct.jenkinsfile'
             load '../ci/stage-test-sandstone.jenkinsfile'
-            load '../ci/stage-test-stress-ng.jenkinsfile'
-            
+            load '../ci/stage-test.jenkinsfile'
+            load '../ci/stage-test-stress-ng.jenkinsfile'
```
Comment (jinengandhi-intel) [inline ci/linux-direct-centos8.2-gcc-release.jenkinsfile:44]: what is this change for? generally unrelated refactoring should be done in a separate PR.

File: ci/linux-sgx-centos8-docker-gcc-release.jenkinsfile [hunk 3]
```diff
@@ -46,8 +46,8 @@ node (node_label) {
                 load '.ci/lib/config-release.jenkinsfile'
 
                 load '../ci/stage-build-sgx.jenkinsfile'
-                load '../ci/stage-test.jenkinsfile'
                 load '../ci/stage-test-sgx.jenkinsfile'
+                load '../ci/stage-test.jenkinsfile'
```
Comment (jinengandhi-intel) [inline ci/linux-sgx-centos8-docker-gcc-release.jenkinsfile:50]: what is this change for? generally unrelated refactoring should be done in a separate PR.

File: ci/linux-sgx-centos8.2-gcc-release.jenkinsfile [hunk 4]
```diff
@@ -40,8 +40,8 @@ node (node_label) {
             load '.ci/lib/config-release.jenkinsfile'
 
             load '../ci/stage-build-sgx.jenkinsfile'
-            load '../ci/stage-test.jenkinsfile'
             load '../ci/stage-test-sgx.jenkinsfile'
+            load '../ci/stage-test.jenkinsfile'
```
Comment (jinengandhi-intel) [inline ci/linux-sgx-centos8.2-gcc-release.jenkinsfile:44]: what is this change for? generally unrelated refactoring should be done in a separate PR.

File: ci/stage-test-examples.jenkinsfile [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+    } catch (Exception e) {
+        env.build_ok = false
+        sh 'echo "Openjdk Example Test Failed"'
+    }
+
+    /*
+    try {
+        timeout(time: 15, unit: 'MINUTES') {
+            sh '''
+                cd CI-Examples/tensorflow-lite
```
Comment (jinengandhi-intel) [inline ci/stage-test-examples.jenkinsfile:39]: didn't you say tf is not working?

**Review by jinengandhi-intel — CHANGES_REQUESTED** [review]

File: ci/linux-direct-centos8-docker-gcc-release.jenkinsfile (hunk 1 above)
Comment (anjalirai-intel) [reply ci/linux-direct-centos8-docker-gcc-release.jenkinsfile:49]: stage-test.jenkinsfile has workload_parser in the end which parses all the workload results in the end. Since CI examples has separate file, that's why this code was needed at the end.

File: ci/linux-direct-centos8.2-gcc-release.jenkinsfile (hunk 2 above)
Comment (anjalirai-intel) [reply ci/linux-direct-centos8.2-gcc-release.jenkinsfile:44]: stage-test.jenkinsfile has workload_parser in the end which parses all the workload results in the end. Since CI examples has separate file, that's why this code was needed at the end.

File: ci/linux-sgx-centos8-docker-gcc-release.jenkinsfile (hunk 3 above)
Comment (anjalirai-intel) [reply ci/linux-sgx-centos8-docker-gcc-release.jenkinsfile:50]: stage-test.jenkinsfile has workload_parser in the end which parses all the workload results in the end. Since CI examples has separate file, that's why this code was needed at the end. So to generalize this piece of code is added at end of jenkinsfile

File: ci/stage-test-examples.jenkinsfile (hunk 5 above)
Comment (anjalirai-intel) [reply ci/stage-test-examples.jenkinsfile:39]: It has been commented and result verification is marked as skip in workload_parser. I didn't want to lose the changes for this workload because we will fix this issue sooner

File: ci/linux-direct-centos8.2-gcc-release.jenkinsfile (hunk 2 above)
Comment (jinengandhi-intel) [reply ci/linux-direct-centos8.2-gcc-release.jenkinsfile:44]: Anjali, I was just saying why change centos files now, when we are not enabling workloads for cent.

Comment (anjalirai-intel) [reply ci/linux-direct-centos8.2-gcc-release.jenkinsfile:44]: I just did to generalize it among all jenkinsfile

**Review by jinengandhi-intel — APPROVED** [review]

**Review by aniket-intelx — APPROVED** [review]
Looks good to me
